logo.js

```diff
@@ -3693,4 +3693,47 @@ function LogoInterpreter(turtle, stream, savehook)
       throw new Error("Internal error: Unbound procedure");
     return value;
   });
+
+  //----------------------------------------------------------------------
+  //
+  // 9. Sound / Basic Music / Web Speech
```

Comment:
These section numbers intentionally match the UCBLogo docs: https://people.eecs.berkeley.edu/~bh/downloads/ucblogo.pdf - just remove the number here.